foundation/src/main/java/org/cbio/portal/pipelines/foundation/FoundationFileTasklet.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2015 Memorial Sloan-Kettering Cancer Center.
+ * Copyright (c) 2016-17 Memorial Sloan-Kettering Cancer Center.
```

Comment:
there is here https://github.com/angelicaochoa/fmi-converter/blob/a59b5d6c51311ffaae188d1af780d7fd7e61f869/foundation/src/main/java/org/cbio/portal/pipelines/foundation/BatchConfiguration.java#L109